skimage/filters/tests/test_thresholding.py

```diff
@@ -561,11 +571,26 @@ def test_multiotsu_output():
     for coor, val in zip(coords, values):
         rr, cc = circle(coor[1], coor[0], 20)
         image[rr, cc] = val
-    thresholds = [64, 128]
+    thresholds = [1, 128]
```

Comment:
I do not understand why this value changed. I would have thought that the example was designed to get a threshold between 64 and 128, and another one between 128 and 192.

Reply:
Hi @sciunto, thank you for your review!
The example image contains 4 values: 0, 64, 128 and 192. Both thresholds are valid:
```python
for ax, thresh in zip(plt.subplots(1, 2)[1], [[1, 128], [64, 128]]): 
    regions = np.digitize(image, thresh) 
    ax.imshow(regions, cmap='Accent') 
    ax.set_title("Thresholds = {}".format(thresh)) 
plt.tight_layout()  
```
![Figure_1](https://user-images.githubusercontent.com/3438227/66141574-2b2ae580-e604-11e9-8c6a-5a3b378ddcaa.png)

Reply:
Then, shouldn't increase the number of bins by one?

Reply:
After some more investigations, I would say that the actual implementation gives the best result: The returned thresholds [t_0, t_1, ..., t_n] separating the classes C_0, C_1, ...., C_(n+1) are defined such as
- C_0 = {x; x ≤ t_0}
- C_1 = {x; t_0 < x ≤ t_1}
- ....
- C_n+1 = {x; x ≤ t_n}

This means that the `right` parameter of `np.digitise` has to be set to `True`:
```python
for ax, thresh in zip(plt.subplots(1, 2)[1], [[1, 128], [64, 128]]): 
    regions = np.digitize(image, thresh, right=True) 
    ax.imshow(regions, cmap='Accent') 
    ax.set_title("Thresholds = {}".format(thresh)) 
plt.tight_layout()  
```
![Figure_3](https://user-images.githubusercontent.com/3438227/66143644-d4bfa600-e607-11e9-893a-f5a420bb840f.png)

The threshold [64, 128] merges the pixels values 0 and 64 making the first circle disappear!
With the thresholds set to [1, 128] the values 64 and 128 are considered as one class and no circle are lost!

If the number of classes is set to 4, both implementations return [0, 64, 128] and all the circles are separated!

![Figure_2](https://user-images.githubusercontent.com/3438227/66144260-cb830900-e608-11e9-818f-119e422a3f74.png)

Reply:
> Then, shouldn't we increase the number of bins by one?

There is no reason for modifying the output since it verify the expected theoretical result.

Reply:
Ah, sorry @sciunto, I got you wrong... You meant increasing the number of bins in the test, right? Yes, of course it is a good idea ;-)

Reply:
> Ah, sorry @sciunto, I got you wrong... You meant increasing the number of bins in the test, right? Yes, of course it is a good idea ;-)

Yes, that's what I meant :)

Reply:
It's done ;-)